src/components/KPagination/KPagination.vue

```diff
@@ -354,12 +433,75 @@ watch(pageCount, (newVal, oldVal) => {
   padding: var(--kui-space-20, $kui-space-20);
   width: 100%;
 
-  .pagination-text {
+  &.full {
+    flex-direction: column;
```

Comment:
I think this is the correct place: on the small viewports where it's full width, can we remove the `4px` of padding at least on the left and right? 

I'm not sure why it's there at all TBH but maybe there's a reason
![image](https://github.com/user-attachments/assets/d3ac2cbd-0238-478e-b8bb-874d2ee5de89)

Reply:
Since there is no keyboard navigation on mobile, meaning the box-shadow (I don't think) would be visible, can we just remove the left and right padding at viewports below `$kui-breakpoint-phablet` ?

Reply:
Done